AGS_Adapter/Convert/Units/Units.cs

```diff
@@ -106,6 +106,11 @@ public static double Units(double value, string unit, string key = "")
                     return value.FromSiemensPerCentimetre()*Math.Pow(10,-6);
                 case "ms/cm":
                     return value.FromSiemensPerCentimetre() * Math.Pow(10, -3); ;
+                // Pressure
+                case "kPa":
+                case "kpa":
+                case "KPA":
```

Comment:
```suggestion
                case "kpa":
```
Only lowercase version required as all white space is removed and `.ToLower()` is used.